thirdparty/runtime_environments_test.go

```diff
@@ -125,3 +125,35 @@ func TestGetImageDiff(t *testing.T) {
 	require.NoError(t, err)
 	assert.Empty(result)
 }
+
+func TestGetHistory(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	assert := assert.New(t)
+	config := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, config, "TestGetHistory")
+	c := NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+
+	// Verify that getHistory errors when not provided the required distro field.
+	_, err := c.getHistory(ctx, DistroHistoryFilterOptions{})
+	assert.Error(err)
+
+	// Verify that getHistory provides images for a distribution.
+	imageNames, err := c.getImageNames(ctx)
+	require.NoError(t, err)
+	require.NotEmpty(t, imageNames)
+	result, err := c.getHistory(ctx, DistroHistoryFilterOptions{Distro: imageNames[1]})
```

Comment:
rather than query the `getImageNames` function, could we just specify a specific image (e.g. `ubuntu2204`)?